theme.ts

```diff
@@ -0,0 +1,45 @@
+const colors = require("tailwindcss/colors");
+import { generateColorRange } from "./common/generateColors";
+
+const primary = generateColorRange("#4E17FF");
+
+export default {
+  darkMode: "selector",
+  theme: {
+    extend: {
+      colors: {
+        black: colors.black,
+        white: colors.white,
+        primary: primary,
+        "page-background": colors.white,
+        "card-border-color": colors.black,
+        "card-background": colors.white,
+        "accordion-header-color": colors.gray["800"],
+        "accordion-header-background": colors.gray["100"],
+        "accordion-header-active-color": colors.black,
+        "accordion-header-active-background": colors.blue["100"],
+        "step-active-border-color": colors.green["500"],
+        "button-text-color": colors.gray["50"],
+        "button-border-color": colors.black,
+        "button-bg-color": primary["500"],
+        "button-bg-hover": primary["600"],
+        "button-bg-focus": primary["600"],
+        "button-bg-active": primary["700"],
+      },
+      boxShadow: {
+        "card-shadow":
+          "0 1px 3px 0 rgb(0 0 0 / 0.1), 0 1px 2px -1px rgb(0 0 0 / 0.1)",
+        "button-shadow":
+          "0 1px 3px 0 rgb(0 0 0 / 0.1), 0 1px 2px -1px rgb(0 0 0 / 0.1)",
+      },
+      borderWidth: {
+        "card-border-width": "1px",
+        "button-border-width": "0px",
+      },
+      borderRadius: {
+        "card-radius": "0.25rem",
+        "button-radius": "0.25rem",
+      },
+    },
+  },
+};
```

Comment:
We don't want to check this file in.

So, this is a bit of an issue with our current development workflow for the installer. Ideally, we could .gitignore it, but in the forks we _do_ need to check the file in, so it's there to be loaded in the vercel deployment.